inmap/cmd/config.go

```diff
@@ -125,6 +128,7 @@ func ReadConfigFile(filename string) (config *ConfigData, err error) {
 	config.InMAPData = os.ExpandEnv(config.InMAPData)
 	config.VariableGridData = os.ExpandEnv(config.VariableGridData)
 	config.OutputFile = os.ExpandEnv(config.OutputFile)
+	config.Logfile = os.ExpandEnv(config.Logfile)
```

Comment:
... if config.LogFile == "". Also, it should probably be LogFile instead of Logfile to be consistent with OutputFile.